client/app/lib/styles/MaterialSummernote.scss

```diff
@@ -1134,8 +1134,8 @@
     background: #ffffff;
     border-color: #e4e4e4;
     margin: 0;
-    padding: 10px 0 15px;
-    text-align: center;
+    padding: 10px 0 15px 15px;
```

Comment:
Any reasons ? I think the shorthand will make the modification easier and final CSS file size smaller.

https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#shorthand
Also checked the docs, looks like it's a configuration issue, scss-lint actually allows this.